src/frontend/komponenter/Behandling/VedtakOgBeregning/InnvilgeVedtak/InnvilgeVedtak.tsx

```diff
@@ -70,6 +72,30 @@ export const InnvilgeVedtak: React.FC<{
             : [tomInntektsperiodeRad],
     });
 
+    const beregnPerioder = useCallback(
+        (vedtaksperiodeListe: IVedtaksperiode[], inntektsperiodeListe: IInntektsperiode[]) => {
+            axiosRequest<IBeløpsperiode[], IBeregningsrequest>({
+                method: 'POST',
+                url: `/familie-ef-sak/api/beregning/`,
+                data: {
+                    vedtaksperioder: vedtaksperiodeListe,
+                    inntekt: inntektsperiodeListe.map((v) => ({
+                        ...v,
+                        forventetInntekt: v.forventetInntekt ?? 0,
+                        samordningsfradrag: v.samordningsfradrag ?? 0,
+                    })),
+                },
+            }).then((res: Ressurs<IBeløpsperiode[]>) => settBeregnetStønad(res));
+        },
+        [axiosRequest]
+    );
+
+    useEffect(() => {
+        if (!behandlingErRedigerbar && lagretInnvilgetVedtak) {
+            beregnPerioder(lagretInnvilgetVedtak.perioder, lagretInnvilgetVedtak.inntekter);
```

Comment:
Bør vi beregne på nytt eller bør vi hente ut den lagrede ytelsen? 
Tenker hvis vi får endringer i beregningsregler i "fremtiden" så kan det bli misvisende å beregne på nytt for gamle? Eller er det en søkt problemstilling?

Reply:
Jodå, enig att vi borde bruke den lagrede ytelsen